forem/forem
# Introduce ProfileFieldGroup model

## What type of PR is this? (check all applicable)


## Description

Based on recent discussions we decided to make the profile groups their own model. This PR 

1. adds the model,
2. updates `ProfileField` to use the new model,
3. Updates/adds specs for the same, and
4. adds a data update script to migrate the existing profile fields.

## Related Tickets & Documents

Part of #6994

## QA Instructions, Screenshots, Recordings

1. Run the migrations locally
2. Run the data update script
3. Verify that all profile fields are now in the correct groups

## Added tests?


## Added to documentation?
